alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java

```diff
@@ -74,19 +74,25 @@ public AlluxioInterpreter(Properties property) {
 
     alluxioMasterHostname = property.getProperty(ALLUXIO_MASTER_HOSTNAME);
     alluxioMasterPort = property.getProperty(ALLUXIO_MASTER_PORT);
+
+
+  }
+
+  private Stream<String> filteredProperties(String prefix) {
+    return properties.stringPropertyNames().stream().filter(propertyKey -> propertyKey.startsWith(prefix));
   }
 
   @Override
   public void open() {
     logger.info("Starting Alluxio shell to connect to " + alluxioMasterHostname +
         " on port " + alluxioMasterPort);
+    // Setting the extra parameters being set in the interpreter config starting with alluxio
+    filteredProperties("alluxio.").forEach(x->System.setProperty(x, properties.getProperty(x)));
```

Comment:
I'm not sure why our checkstyle couldn't check it but, 
```suggestion
    filteredProperties("alluxio.").forEach(x -> System.setProperty(x, properties.getProperty(x)));
```